qutebrowser/browser/network/qutescheme.py

```diff
@@ -159,22 +159,37 @@ def qute_version(_win_id, _request):
 
 @add_handler('plainlog')
 def qute_plainlog(_win_id, _request):
-    """Handler for qute:plainlog. Return HTML content as bytes."""
+    """Handler for qute:plainlog. Return HTML content as bytes.
+
+    An optional query parameter specifies the min log level to print.
+    For example, qute://log?level=warning prints warnings and errors.
+    Level can be one of: vdebug, debug, info, warning, error, critical.
+    """
     if log.ram_handler is None:
         text = "Log output was disabled."
     else:
-        text = log.ram_handler.dump_log()
+        query = QUrlQuery(_request.url().query())
+        level = query.queryItemValue('level')
+        text = log.ram_handler.dump_log(html=False, level=level)
     html = jinja.render('pre.html', title='log', content=text)
     return html.encode('UTF-8', errors='xmlcharrefreplace')
 
 
 @add_handler('log')
 def qute_log(_win_id, _request):
-    """Handler for qute:log. Return HTML content as bytes."""
+    """Handler for qute:log. Return HTML content as bytes.
+
+    An optional query parameter specifies the min log level to print.
+    For example, qute://log?level=warning prints warnings and errors.
+    Level can be one of: vdebug, debug, info, warning, error, critical.
+    """
     if log.ram_handler is None:
         html_log = None
     else:
-        html_log = log.ram_handler.dump_log(html=True)
+        query = QUrlQuery(_request.url().query())
```

Comment:
Please rename the `_request` argument to `request` (as `_foo` is the notation for "unused argument")